examples/jsm/nodes/accessors/SkinningNode.js

```diff
@@ -78,6 +78,16 @@ class SkinningNode extends Node {
 
 	}
 
+	generate( builder, output ) {
+
+		if ( output !== 'void' ) {
+
+			return positionLocal.build( builder, output );
```

Comment:
Couldn't we just return `positionLocal` in `setup`? Or would it not work in this way?

Reply:
It will not work if the `output` is `void`, but this treatment can certainly be done automatically soon.